app/controllers/organizations_controller.rb

```diff
@@ -225,4 +225,3 @@ def verify_user_belongs_to_organization
     not_found unless current_organization.users.map(&:id).include?(@removed_user.id)
   end
 end
-# rubocop:enable Metrics/ClassLength
```

Comment:
This didn't seem to have a partner that was disabling anything. No linters failed when I removed it either.